### DISCUSSION TITLE
HyperSwitch Widget Needs Saved Cards Configuratble

### DISCUSSION THREAD

**Discussion by nara-golfdistrict-com** [juspay/hyperswitch#4212] [Ideas / Feature Requests]
See attached screenshot.
Currently when we have many several cards, the totals and any other information that we have under the HyperSwitch payment widget will get pushed down thus making the info not visible in the first screen and the user has to scroll.

I propose that we make the saved cards selectable via a dropdown instead of showing them individually using a radio button. This will occupy less space.

![image](https://github.com/juspay/hyperswitch/assets/145275696/fd7cf449-54e1-471d-a474-0ffdc5ba1be2)

**Comment by akash-c-k** [comment]
hey @nara-golfdistrict-com , thank you for the feedback. I'll discuss this with our designers and see how best to handle this! I was curious why you have the important details below the checkout iframe? You can have all the important details like amount, etc. at the top and then give iframe space to the SDK.

**Comment by nara-golfdistrict-com** [comment]
@akash-kamble-juspay That's exactly what we are going to do right now to move the other important details like totals, etc. to the top.
When we originally designed the page, we didn't realize that having many cards would push the screen out of the viewport. Also at that time the saved cards functionality was not there.

Keep in mind, even if we move the important details like the totals to the top, the "Buy" button is still going to be below the screen. That's why it is important for HyperSwitch to consider changing the design to give more real estate.
